### ISSUE TITLE
Version Detection is not working

### ISSUE THREAD

**Issue by dev2214** [browser-update/browser-update#408]
Hi I have put configuration as below:

required:{e:0,f:0,o:0,s:0,c:0},

According to API, alert should appear except latest version.
but for chrome and firefox does not appear.

Can you please help me here.

**Comment by josselex** [comment]
This should work. Only the version before the latest is also not notified
for some weeks as a grace period. Otherwise a lot of people would be
notified because the automatic updates are usually rolling out slowly.

Am Fr., 27. Juli 2018 um 17:18 Uhr schrieb dev2214 <notifications@github.com

**Comment by dev2214** [comment]
Hi 
I have tried with latest version but still no luck.
I noticed that when we keep 0 it avoids and it takes what is in JS file.
one more issue i found that after appearing alert box if i do refresh the browser it calls the ignore action and 2nd time the box does not appear.

My configuration is as below.
var $buoop = {required:{e:-0,f:0,o:0,s:0,c:0},insecure:true,api:2018.08,
reminderClosed:336,
text:{
	msg:'Note: You are using a browser designed for older applications. This site works best with current versions of Chrome, Edge, Firefox, or Safari.',
	bignore: 'Ignore',
        bupdate: '',
msgmore:'',
	remind: 'You will be reminded in {days} days.',
	bnever: 'Never Show Again'   
} }; 
function $buo_f(){ 
 var e = document.createElement("script"); 
 e.src = "//browser-update.org/update.min.js"; 
 document.body.appendChild(e);
};
try {document.addEventListener("DOMContentLoaded", $buo_f,false)}
catch(e){window.attachEvent("onload", $buo_f)}
